sdk/core/Azure.Core/api/Azure.Core.net6.0.cs

```diff
@@ -518,6 +519,11 @@ public static partial class MultipartResponse
         public static Azure.Response[] Parse(Azure.Response response, bool expectCrLf, System.Threading.CancellationToken cancellationToken) { throw null; }
         public static System.Threading.Tasks.Task<Azure.Response[]> ParseAsync(Azure.Response response, bool expectCrLf, System.Threading.CancellationToken cancellationToken) { throw null; }
     }
+    public partial class RedirectPolicyOptions
+    {
+        public RedirectPolicyOptions() { }
+        public bool IsAutoRedirectEnabled { get { throw null; } set { } }
```

Comment:
I would call it IsClientRedirectEnabled to hint that it does not control networking stack redirects but just the code of the client.